index.html

```diff
@@ -388,5 +389,30 @@ <h2>About me</h2>
             </div>
         </section>
     </main>
+    <footer class="footer">
+        <div class="footerHeader">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        </div>
+  
+        <div class="footerForm">
+            <form action="https://formspree.io/f/mgedowaj" method="POST">
+                <input class="ftext" type="text" id="Full-Name" name="name" placeholder="Full name" maxlength="30" required>
+                <input class="femail" type="email" id="Email" name="_replyto" placeholder="Email address" required>
```

Comment:
- [ ] Please kindly correct these form fields as they have not been corrected as the previous reviewers requested.  I tested your project on Windows, and Linux, using Chrome and Firefox.  

Your project:

![image](https://user-images.githubusercontent.com/57421823/149060837-20200023-221e-412e-81c9-857b4f27c199.png)

The Figma:

![image](https://user-images.githubusercontent.com/57421823/149061010-0cc39905-4302-4efc-9829-b07c60213c14.png)

You can use the properties given in the template in order to figure out how to do it:

![image](https://user-images.githubusercontent.com/57421823/149061230-9aabe081-03b1-488e-aea6-e3dc2acf3ee6.png)

Let's correct these form fields so your project looks like the given design 💪